mso/resource_mso_schema_site_anp_epg_domain.go

```diff
@@ -235,8 +289,7 @@ func resourceMSOSchemaSiteAnpEpgDomainImport(d *schema.ResourceData, m interface
 									d.SetId(apiDomain)
 									d.Set("site_id", apiSite)
 									d.Set("domain_type", models.StripQuotes(domainCont.S("domainType").String()))
-									d.Set("dn", domain)
-									d.Set("deployment_immediacy", models.StripQuotes(domainCont.S("deployImmediacy").String()))
+									d.Set("deploy_immediacy", models.StripQuotes(domainCont.S("deployImmediacy").String()))
```

Comment:
Shouldn't we also have domain_dn, domain_name and vmm_domain_type here?

Reply:
I added domain_dn. The other attributes are already set at the beginning of import.

Reply:
but is there any reason to set them until this place?

Reply:
domain_name and vmm_domain_type are taken from user input(terraform import). If I add those two here, they will be in nested loop and executed multiple times without reason.